index.html

```diff
@@ -349,67 +349,94 @@ <h3>Use Cases and Requirements</h3>
 desirable ecosystem characteristics were identified for this specification,
 specifically:
         </p>
-
-        <ul>
+<!-- requirement list start -->
+        <ol>
           <li>
-<a>Holders</a> receive and store <a>verifiable credentials</a> from
-<a>issuers</a>. <a>Holders</a> might interact with an <a>issuer</a> through an
-agent that the <a>issuer</a> does not need to trust.
+<a>Verifiable Credentials</a> represent statements made by an <a>Issuer</a> in
+a tamper evident and privacy respecting manner.
           </li>
           <li>
-<a>Holders</a> are positioned between <a>issuers</a> and <a>verifiers</a>
-and use <a>verifiable credentials</a> to make <a>verifiable presentations</a>
-to <a>verifiers</a>.
+<a>Holders</a> assemble collections of credentials from different issuers into a
+single artifact, a <a>Verifiable Presentation</a>.
           </li>
           <li>
-<a>Holders</a> might interact with <a>verifiers</a> through an agent that
-<a>verifiers</a> do not need to trust; <a>verifiers</a> only need to trust
-<a>issuers</a>.
+<a>Issuers</a> can issue <a>Verifiable Credentials</a> about any <a>subject</a> .
           </li>
           <li>
-<a>Verifiable credentials</a> are associated with <a>subjects</a>, not
-specific services; <a>holders</a> decide how to aggregate and manage
-<a>verifiable credentials</a> and the <a>verifiable presentations</a>
-associated with them.
+Acting as <a>Issuer</a>, <a>Holder</a>, or <a>Verifier</a> requires neither registration
+nor approval by any authority.
           </li>
           <li>
-<a>Subjects</a> can easily acquire and control their identifiers.
+<a>Verifiable Presentations</a> allow any <a>Verifier</a> to verify the authenticity
+of credentials from any <a>Issuer</a>.
           </li>
           <li>
-<a>Holders</a> control which <a>verifiable credentials</a> to use and when.
+<a>Holders</a> can receive <a>Verifiable Credentials</a> from anyone.
           </li>
           <li>
-<a>Holders</a> can freely choose and change the agents they employ to help
-them manage their <a>verifiable credentials</a> and generate and share
-<a>verifiable presentations</a>.
+<a>Holders</a> can interact with any <a>Issuer</a> and any <a>Verifier</a> through
+any User Agent.
           </li>
           <li>
-<a>Holders</a> can share <a>verifiable presentations</a> that can be verified
-without revealing the identity of the <a>verifier</a> to the <a>issuer</a>.
+<a>Holders</a> can store <a>Verifiable Credentials</a> in any location, without
+affecting their verifiability and without the <a>Issuer</a> knowing anything about
+where they are stored or when they are accessed.
           </li>
           <li>
-<a>Holders</a> in <a>presentations</a> can either disclose the attributes or
-satisfy <a>derived predicates</a> requested by the <a>verifier</a>.
```

Comment:
I'd like derived predicates to remain in the list. 
It is an important feature that not only may a presentation not contain the entire credential (selective disclosure of claims), but also only reveal a predicate based on the original value rather than the value itself. I think "derived predicate" is the most accurate description of this (it is a predicate derived from a claim).